src/routes/+layout.svelte

```diff
@@ -48,11 +48,11 @@
 					height: auto;
 				}
 
-				p {
-					font-family: 'Lilita One', sans-serif;
-					color: $brown;
-					font-size: 2rem;
-				}
+				// p {
```

Comment:
I don't think we need this, you can just delete the entire commented code block.